examples/structured_2d_dgsem/elixir_advection_parallelogram.jl

```diff
@@ -25,6 +25,7 @@ end
 ###############################################################################
 # semidiscretization of the linear advection equation
 
+# Transformed advection_velocity = (0.2, -0.7) by transformation mapping
 advection_velocity = (-0.5, -0.7)
```

Comment:
`(0.2, -0.7)` vs. `(-0.5, -0.7)`?

Reply:
This is the transformation mapping applied to `(0.2, -0.7)`. This elixir should have the same nodal values of the IC and exact solution, just with different metric terms. That way, it produces the exact same results as the `elixir_advection_basic`.
In fact, I just added this explanation to the elixir in another branch while reworking the structured and p4est elixirs.

Reply:
Okay, I'm looking forward to it